generic3g/specs/make_itemSpec.F90

```diff
@@ -65,4 +74,29 @@ function make_itemSpec(variable_spec, registry, rc) result(item_spec)
 
    end function make_itemSpec
 
+
+   subroutine split_name(encoded_name, name_1, name_2, rc)
+      character(*), intent(in) :: encoded_name
+      character(:), allocatable, intent(out) :: name_1
+      character(:), allocatable, intent(out) :: name_2
+      integer, optional, intent(out) :: rc
+
+      integer :: status
+      integer :: idx_open, idx_close, idx_comma
+
+      idx_open = index(encoded_name, '(')
+      idx_close = index(encoded_name, ')')
+      idx_comma = index(encoded_name, ',')
+
+      _ASSERT(idx_open > 0, 'VectorAspect requires standard name to have tuple for east west components')
```

Comment:
Is it always "east west"?

Reply:
No.  But it will be a separate feature to support the alternate orientation.   Low priority for the moment.    But I will at least make the comment more generic.